README.md

```diff
@@ -24,9 +24,18 @@ There are no dependencies.
 
 ## Usage
 
-Below are sample usages for the playbook:
+Currently we have provided 2 ways to retrive this role:
+ * From Ansible-Galaxy
+ * Clone from this repo
+
+### Download From Ansible-Galaxy
```

Comment:
Download From Ansible-Galaxy -> Download from Ansible-Galaxy.
Please use 'sentence case' for all headings. Example:

Correct: Download your file now
Incorrect: Download Your File Now

Even though you are using sentence case, remember to preserve capital letters on things that require a capital letter.
Correct: Download the Agent now
Incorrect: Download the agent now